src/flexbuffer_json.h

```diff
@@ -672,27 +647,19 @@ class JsonObject : JsonWithPath
         double get_float( const char *key, double fallback ) const;
```

Comment:
Migrate these accessors too

Reply:
I had planned to do those in a pater PR, but I guess since they are sandwiched between two other groups I migrated I should probably do them now.